### PR TITLE
adding 3 files

### PR DESCRIPTION
**Pull request by oleh-stekh**
DEMO LINK:https://oleh-stekh.github.io/layout_html-form/
TEST REPORT LINK:https://oleh-stekh.github.io/layout_html-form/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,14 +4,172 @@
...
     <script type="text/javascript" src="./main.js"></script>
+    <form 
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+    method="post"
+    >
+      <fieldset class="formFieldset">
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input
```
Comment (mykhalenych) [inline src/index.html:23]: Screenshot 2023-01-17 at 17 24 24

Comment (mykhalenych) [reply src/index.html:23]: check all elements

File: src/index.html [hunk 2]
```diff
@@ -4,14 +4,172 @@
...
+      </fieldset>
+      <fieldset class="formFieldset">
+        <legend>An interesting fact about you!</legend>
+        <label class="radioCatLovers">
+          Do you love cats?
+          <input 
+          type="radio" 
+          name="catLover" 
+          value="yes" 
+          required
```
Comment (mykhalenych) [inline src/index.html]: why catLover is required?

File: src/index.html [hunk 3]
```diff
@@ -4,14 +4,172 @@
...
+          name="password" 
+          minlength="4" 
+          autocomplete="off" 
+          required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="formFieldset">
+        <legend>An interesting fact about you!</legend>
+        <label class="radioCatLovers">
```
Comment (mykhalenych) [inline src/index.html]: use kebab-case for classnames

File: src/index.html [hunk 4]
```diff
@@ -4,14 +4,172 @@
...
+      content="width=device-width, user-scalable=no, 
+      initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
```
Comment (mykhalenych) [inline src/index.html]: better to add script in the end of body

File: src/style.css [hunk 5]
```diff
@@ -1 +1,14 @@
 /* styles go here */
+.form-field {
+  display: block;
+  margin: auto auto 10px;
```
Comment (mykhalenych) [inline src/style.css]: check styles you failed 6 tests

**Review by mykhalenych — CHANGES_REQUESTED** [review]

**Review by witflash — APPROVED** [review]
Well done 👍🏻

**Pull request closed without merging**